Index: spotify_wrapped/.cache
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\"access_token\": \"BQAuphziyJrT5sQDVvP1ZCSkcF-e5AovXFO5xO5MwVhxkIWNCoVKuGZToiRNNqTmmwIiGdH4syZHBaSeEkWKM5x-41nzOK3p4KahnhL6k4fuyTIIj8GHes592q9VPWdYm0bw9NAnyZmgGHpXOjRG8j0G6F8CA6Q6zD5K84PwHZcit4dJfwcgOibPCwZOtsUC2dohY5tEnfhXsd18BZthXmRdKm4gKdwonndlVGQ\", \"token_type\": \"Bearer\", \"expires_in\": 3600, \"refresh_token\": \"AQC4Y4tUxX2KrmQpwK-QwGWRh4l9ofoeyCc8i9hsIfBJWIdZ0isZ7t-1jJa5vrpozJ4SzqosDIFgDsw68rIqTEUOozFJlVdifBiXgzQl94fh_1dskl-WZ2eBPOwyp67xIY4\", \"scope\": \"user-read-private user-read-email user-top-read\", \"expires_at\": 1732049724}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/spotify_wrapped/.cache b/spotify_wrapped/.cache
--- a/spotify_wrapped/.cache	(revision cc0307cab690c1c3e74e6ae2b976253dc41b37a5)
+++ b/spotify_wrapped/.cache	(date 1732047121636)
@@ -1,1 +1,1 @@
-{"access_token": "BQAuphziyJrT5sQDVvP1ZCSkcF-e5AovXFO5xO5MwVhxkIWNCoVKuGZToiRNNqTmmwIiGdH4syZHBaSeEkWKM5x-41nzOK3p4KahnhL6k4fuyTIIj8GHes592q9VPWdYm0bw9NAnyZmgGHpXOjRG8j0G6F8CA6Q6zD5K84PwHZcit4dJfwcgOibPCwZOtsUC2dohY5tEnfhXsd18BZthXmRdKm4gKdwonndlVGQ", "token_type": "Bearer", "expires_in": 3600, "refresh_token": "AQC4Y4tUxX2KrmQpwK-QwGWRh4l9ofoeyCc8i9hsIfBJWIdZ0isZ7t-1jJa5vrpozJ4SzqosDIFgDsw68rIqTEUOozFJlVdifBiXgzQl94fh_1dskl-WZ2eBPOwyp67xIY4", "scope": "user-read-private user-read-email user-top-read", "expires_at": 1732049724}
\ No newline at end of file
+{"access_token": "BQD0uiQrcV9JHaNIVlN5O3eI18GaSLtGczxt3ekZx1r0XYDdkVPQ3TAVHtGxzvZb4UuRWhVRBiUQI7TD32xhTkIEYzNc73CEEY3sQSDYAD6eWwV7bhC_xpRHWUNs3N2xZJG4qq_soLmu51OO_nurTjZ-LQOt8Ak_ZBdGBU9ytOyTfFl3rDyIYjPAdWM0E0MIV_B7E8a8ThjL6l5Un0GICl1Q1DQGYvY1hLyPwpA", "token_type": "Bearer", "expires_in": 3600, "refresh_token": "AQBnpYu49ro74ede4wcVTnRRhXtwOUrDIDzeHjrlFRAlRXv00pGdjilOBhnwtp26_fug7bil5TPJ330JveeZ5R0LXqwvMtC2yaVYEoQlBVZYrGOM9D5fYSl2VVZyWYiS_rQ", "scope": "user-read-private user-read-email user-top-read", "expires_at": 1732050721}
\ No newline at end of file
